src/common/transaction-confirmation/index.tsx

```diff
@@ -162,6 +163,12 @@ const TransactionConfirmation = ({ shouldShow, handleClose, transaction, to, fro
   const [toPrice] = useRawUsdPrice(to);
   const protocolToken = getProtocolToken(currentNetwork.chainId);
   const [protocolPrice] = useRawUsdPrice(protocolToken);
+  const trackEvent = useTrackEvent();
+
+  const handleNewTrade = () => {
+    trackEvent('Aggregator - New trade');
+    handleClose();
+  };
```

Comment:
Is this the button to got for a new trade once you are on the receipt page?

Reply:
Exactly